Podfile.lock

```diff
@@ -20,6 +21,7 @@ PODS:
 DEPENDENCIES:
   - Expecta
   - HockeySDK (~> 3.6.2)
+  - Keys (from `/Users/ash/bin/Tropos/Pods/CocoaPodsKeys/`)
```

Comment:
Hey, good catch! This bug was actually just [reported today](https://github.com/orta/cocoapods-keys/issues/92). I'll update the PR with the fixed version.